documentation/src/docs/deployment/run-local.md

```diff
@@ -60,23 +60,61 @@ docker run -it -e HOST_IP=<ip> -p 9090:9090 -p 12345:12345 pravega/pravega:lates
 
 ## Docker Compose (Distributed Mode)
 
-Unlike other options for running locally, the docker compose option runs a full deployment of Pravega
+Unlike other options for running locally, the Docker Compose option runs a full deployment of Pravega
 in distributed mode. It contains containers for running Zookeeper, Bookkeeper and HDFS. Hence Pravega operates as if it would in production. This is the easiest way to get started with the standalone option but requires additional resources.
 
-**Prerequisite:** Docker `1.12` or later.
+1. Before attempting to deploy Pravega using this option, be sure your host machine meets the following prerequisites:
 
-Download the [docker-compose.yml](https://github.com/pravega/pravega/tree/master/docker/compose/docker-compose.yml) from github. For example:
+   * It has Docker `1.12` or later installed.
+   * It has Docker Compose installed.
+   * Download the [docker-compose.yml](https://github.com/pravega/pravega/tree/master/docker/compose/docker-compose.yml) from github. For example:
 
-```
-wget https://raw.githubusercontent.com/pravega/pravega/master/docker/compose/docker-compose.yml
-```
+   ```
+   $ wget https://raw.githubusercontent.com/pravega/pravega/master/docker/compose/docker-compose.yml
+   ```
 
-We need to set the IP address of our local machine as the value of `HOST_IP` in the following command.
+   Alternatively, clone the Pravega repository to fetch the code.
+
+   ```
+   $ git clone https://github.com/pravega/pravega.git
+   ```
+
+2. Navigate to the directory containing Docker Compose configuration `.yml` files.
+
+   ```
+   $ cd /path/to/pravega/docker/compose
+   ```
+
+3. Add HOST_IP as an environment variable with the value as the IP address of the host.
+
+   ```
+   $ export HOST_IP=<host-ip>
+   ```
+
+4. Now, run the following command to start a deployment comprising of multiple Docker containers, as specified in the
```

Comment:
@ravisharda can u remove now., run .....to Run the

Reply:
Done.